websites/P/Phind/presence.ts

```diff
@@ -0,0 +1,95 @@
+const presence = new Presence({
+		clientId: "1106990410838065172",
+	}),
+	slideshow = presence.createSlideshow(),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const Assets = {
+			PhindLogo: "https://i.imgur.com/fakWcYA.png",
+			Search: "https://i.imgur.com/a5qsEbL.png",
```

Comment:
Use the build in Search asset. And use const enum Assets to add the PhindLogo, that way you can use the build-in search asset.

Reply:
Oh that's what was meant with
> Use standardized asset
My bad.

Reply:
Updated Search to use `Assets.Search`